Google.Cloud.EntityFrameworkCore.Spanner/Storage/Internal/SpannerDatabaseCreator.cs

```diff
@@ -118,19 +119,12 @@ private IReadOnlyList<MigrationCommand> CreateCreateOperations()
         /// <inheritdoc />
         public override async Task<bool> ExistsAsync(CancellationToken cancellationToken = default)
         {
-            //TODO: this should be fixed up to call an admin method instead of looking for an error code.
             try
             {
-                await CreateHasTablesCommand().ExecuteReaderAsync(new RelationalCommandParameterObject(
-                                        _connection,
-                                        null,
-                                        null,
-                                        Dependencies.CurrentContext.Context,
-                                        Dependencies.CommandLogger), cancellationToken)
-                    .ConfigureAwait(false);
+                DatabaseAdminClient databaseAdminClient = DatabaseAdminClient.Create();
```

Comment:
The `DatabaseAdminClient` needs to be built with the option of connecting to the emulator as well as to production. See https://github.com/cloudspannerecosystem/dotnet-spanner-entity-framework/blob/228615ae8592580db2bfefc3f699818fdf127f4e/Google.Cloud.EntityFrameworkCore.Spanner.Samples/SampleRunner.cs#L171 for an example (that example always connects to the emulator, but the enum that is used has a value that allows `EmulatorOrProduction`).